packages/meta/index.js

```diff
@@ -182,12 +183,12 @@ function generateMeta (_options) {
     options.ogImage = { path: options.ogImage }
   }
   if (options.ogImage && !find(this.options.head.meta, 'property', 'og:image') && !find(this.options.head.meta, 'name', 'og:image')) {
-    if (options.ogHost) {
+    if (options.ogHost || isUrl(options.ogImage.path)) {
```

Comment:
@ricardogobbosouza 

Because of this `og:image` is not added to the head with default settings.
Is this expected?

`options.ogImage` when using default settings (logged before this line).
```ts
{                                                                                                                                                                                           01:51:19
  path: '/_nuxt/icons/icon_512.b491dd.png',
  width: 512,
  height: 512,
  type: 'image/png'
}
```